__docs__/wonder-blocks-dropdown/multi-select.stories.tsx

```diff
@@ -174,6 +182,47 @@ export const Default: StoryComponentType = {
     },
 };
 
+/**
+ * The field can be used with the LabeledField component to provide a label,
+ * description, required indicator, and/or error message for the field.
+ *
+ * Using the field with the LabeledField component will ensure that the field
+ * has the relavent accessibility attributes set.
```

Comment:
A spelling nit (used in a few places): `relavent` -> `relevant`

Reply:
Thanks for catching that! 😄 fixed!